src/main/java/uk/gov/hmcts/reform/coh/controller/onlinehearing/Relisting.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.coh.controller.onlinehearing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.reform.coh.domain.RelistingState;
+
+import javax.validation.constraints.NotNull;
+
+public class Relisting {
```

Comment:
Don't think the Relisting members should be public

Reply:
It is still a data structure, whether it uses setters/getters or public fields.

The `public final` approach doesn't require so much boilerplate code. I am happy to change if missed anything valuable.

Reply:
Exposing the inner members of a class doesn't sit right with me. You can always to auto code generator or Lombok?